### DISCUSSION TITLE
Is there an online walk-thru on how to use Codespaces

### DISCUSSION THREAD

**Discussion by misterskye** [earthlab-education/Earth-Analytics-2023-01-Intro#209] [Q&A02 Open Reproducible Science]
I feel like I'm totally moving in all the wrong directions and am in need of some hand-holding.   

I get that we are suppose to commit changes in Codespaces, but How?
How do we go about creating a new Jupyter notebook?

@eculler --Expect me at your office hours tomorrow :-)

**Comment by eculler** [comment]
Hi Skye,

You can commit and sync your changes in the GitHub tab on the left (3rd from the top I believe).

I will be providing instructions on submitting your analysis in a new location tomorrow (it’s not due until Friday this week so we can go over it in section). In the meantime, to create a new notebook, click the add file button in the file explorer, and then name it something with the extension .ipynb.

Look forward to seeing you tomorrow!

Elsa Culler, PhD Civil Engineering
she/her/hers

Earth Data Science Instructor
Earthlab and ESIIL
CIRES
University of Colorado Boulder

CU Boulder is located on the traditional territories and ancestral homelands of the Cheyenne, Arapaho, Ute, Lakota, and many other Native American nations.

**Reply by eculler** [reply]
Oh, and if you check out the very end of the video for this week's assignment I believe I do show how to commit and sync there. But I'll put up a separate video in the how-to on the wiki and post about it here.

**Comment by misterskye** [comment]
Thank you for the reminder about the video!!  Wondering in the future if the  weeks  'issue' could be put into the readme file of the current coding challenge.

**Comment by edchanonline** [comment]
To create a jupyter notebook, I used the command line within VS code: `$ cp (old_file) (new_file)`. That might be easier so you retain the same codes you have written.
